docs/components/table.md

```diff
@@ -230,6 +230,36 @@ object these features should be explicitly disabled.
 </template>
 ```
 
+### fetcherCacheKey
+
+The fetcher functionality makes use of SWRV to handle caching of response data. Whenever the cache key is changed the fetcher will automatically
```

Comment:
Add a link to `SWRV` docs: https://docs-swrv.netlify.app/

Reply:
Fixed!